derpconf/config.py

```diff
@@ -76,6 +76,16 @@ def load(cls, path, conf_name=None, lookup_paths=[], defaults={}):
 
     @classmethod
     def __load_from_path(cls, conf, path):
+        if isdir(path):
+            conf.config_folder = path
+            files = sorted(os.listdir(path))
```

Comment:
Like, pointing to a config file right now, derpconf doesn't enforce it to end in .conf. I think it would be weird to make it do so for the folder use case.